src/custom/pages/Profile/styled.tsx

```diff
@@ -429,6 +429,25 @@ export const BannerCard = styled(BannerExplainer)`
   }
 `
 
+export const CardActions = styled.div`
+  width: 100%;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
+
+  > a {
```

Comment:
One suggestion, maybe for consistency we can add text color change to orange on `hover` as it works now for the governance card links.

Reply:
Good find, will update in an upcoming PR 👍🏼